components/Footer.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <Link href="/"> 🏋️‍♀️ </Link>
```

Comment:
I would suggest you like to another page here which could be called "my gymplan" or something like that where you display the values which are added by the User in your form. Until now, the workout is never displayed anywhere and this emoji would actually be perfect for that, but then it would need to be included in the form as a submit button and not in the footer :)